### PR TITLE
Workspace: Add Keyboard Events to New Font Picker

### PR DESCRIPTION
**Pull request by carloskelly13**
## Summary

Adds <kbd>Esc</kbd>, <kbd>Enter</kbd>, <kbd>Tab</kbd>, <kbd>Up</kbd>, and <kbd>Down</kbd> navigation support to new font picker.

- <kbd>Esc</kbd> closes the menu
- <kbd>Enter</kbd> selects the current item and sets the font to the text
- <kbd>Tab</kbd> or <kbd>Down</kbd> select the item below
- <kbd>Shift</kbd> + <kbd>Tab</kbd> or <kbd>Up</kbd> selects the previous item

Bounds are used to make sure the user cannot go negative or too high in the list. Adds tests for the navigation events.

## Relevant Technical Choices

Uses a keyboard event handler to manage the actions when a key is pressed.

## User-facing changes

The user can now navigate the font selector using the keyboard.

## Testing Instructions

- Enable the new font picker flag
- Use the keyboard sequences when the new font picker is open as listed above to navigate the picker

---

#2205

### CODE REVIEW

File: assets/src/edit-story/components/fontPicker/pickerContainer.js [hunk 1]
```diff
@@ -132,11 +134,28 @@ function FontPickerContainer({ value, onSelect, onClose }) {
...
   // This is static for now, but with search and used fonts, this will change later
   const matchingFonts = fonts;
 
+  const handleKeyPress = useCallback(
+    ({ nativeEvent: { code, shiftKey } }) => {
+      if (code === 'Escape') {
+        onClose();
+      } else if (code === 'Enter') {
+        onSelect(fonts[currentOffset].name);
+      } else if (code === 'ArrowUp' || (code === 'Tab' && shiftKey)) {
```
Comment (swissspidy) [inline assets/src/edit-story/components/fontPicker/pickerContainer.js]: <kbd>Tab</kbd> support is odd.

This seems like I can never ever get out of the font picker by using <kbd>Tab</kbd>, unless I go through all 1000 items in the list.

On that case, only arrow keys should be supported for going up and down.

Comment (carloskelly13) [reply assets/src/edit-story/components/fontPicker/pickerContainer.js]: That is true. We can drop tab support.

Comment (swissspidy) [reply assets/src/edit-story/components/fontPicker/pickerContainer.js]: cc @barklund to confirm, but IMHO we should drop it.

Comment (carlos-kelly) [reply assets/src/edit-story/components/fontPicker/pickerContainer.js:150]: I have dropped it locally. Should we want this, I can push it up.

Comment (carloskelly13) [reply assets/src/edit-story/components/fontPicker/pickerContainer.js]: Confirmed with Morten to have no tab support. Removed and pushed.

**Comment by carloskelly13** [comment]
Closing this since the changes and review have been folded into #2313

**Pull request closed without merging**